core/templates/pages/skill-editor-page/services/skill-editor-state.service.ts

```diff
@@ -73,7 +73,9 @@ export class SkillEditorStateService {
       // The skill is set directly for the first load.
       this._skill = skill;
     } else {
-      // For every other load.
+      // After first initialization, the skill object will be retained for
+      // the lifetime of the editor and on every reload, the new contents
```

Comment:
What is meant by 'reload' here?

Reply:
It is reloaded in certain scenarios such as when we change the tabs inside the skill editor, the skill loads again from the backend. Similarly, when a change is made in the editor, the skill is updated and the contents are copied to the object.

Reply:
Ah, right, I thought it is talking about page reload. Could we write "data reload" instead?

Reply:
done!